src/api/helpers/mapSicCodes.ts

```diff
@@ -0,0 +1,54 @@
+interface CompanyResponse {
+  id: string | null;
+  applicationId: string | null;
+}
+
+interface ConnectId {
+  id: string;
+}
+
+interface ConnectObj {
+  connect: ConnectId;
+}
+
+interface SicCodes {
+  code: string;
+  exporterCompany: ConnectObj;
+  application: ConnectObj;
+}
+
+/**
+ * maps sic codes from response and adds to an array to add to the database
+ * @param {CompanyResponse} company
+ * @param {Array} sicCodes
+ * @returns {Array} sicCodesToAdd
+ */
+const mapSicCodes = (company: CompanyResponse, sicCodes?: Array<string>) => {
+  const sicCodesToAdd = [] as Array<SicCodes>;
+
+  if (!sicCodes || !sicCodes.length) {
+    return sicCodesToAdd;
+  }
+
+  sicCodes.forEach((code) => {
+    const codeToAdd = {
+      code,
+      exporterCompany: {
+        connect: {
+          id: company.id,
+        },
+      },
+      application: {
+        connect: {
+          id: company.applicationId,
+        },
+      },
+    } as SicCodes;
+
+    sicCodesToAdd.push(codeToAdd);
+  });
+
```

Comment:
I think a map could work more nicely here? 

```js
mapped = sicCodes.map((code) => ({
  code,
  exporterCompany: {},
  // etc
});
```

Should we also rename `sicCodesToAdd` to `mapped`, because it's a mapping funciton, what do you think?

Reply:
done